app/awards/Awards.tsx

```diff
@@ -0,0 +1,62 @@
+import { Wrap, WrapItem } from "@chakra-ui/react";
+import { memo } from "react";
+import { Image } from "~app/image";
+import awardCae from "./award-cae.png";
+import awardCils from "./award-cils.png";
+import awardCourage from "./award-courage.jpg";
+import awardDelf from "./award-delf.jpg";
+
+export default memo(function Awards() {
+	return (
+		<Wrap
+			spacing={8}
+			justify="space-between"
+			w="full"
+			mt="5"
+			maxW="6xl"
+			pos="relative"
+			_before={{
+				bg: "linear-gradient(90deg, rgba(66, 71, 112, 0.09), rgba(66, 71, 112, 0.09) 50%, transparent 0, transparent)",
+				bgSize: "8px 1px",
+				content: "''",
+				h: "1px",
+				left: "0",
+				pos: "absolute",
+				top: "-20px",
+				w: "full",
+			}}>
+			<WrapItem h="70px">
```

Comment:
I wonder if it's possible to fully rely on the `<Wrap>`'s `spacing` instead of applying a height to each item individually...